.gitignore

```diff
@@ -823,3 +823,4 @@ FodyWeavers.xsd
 # Additional files built by Visual Studio
 
 # End of https://www.toptal.com/developers/gitignore/api/c,c++,cmake,linux,macos,windows,autotools,visualstudio,visualstudiocode,jetbrains
+_codeql_*
```

Comment:
Was it you that added that? I never saw this pattern ignored yet, and I have codeql running in my repos

Reply:
Its justified then, keep it